FinniversKit/Sources/Cells/RecentlyFavoritedShelfCell/RecentlyFavoritedViewModel.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+public struct RecentlyFavoritedViewmodel: Hashable{
+    public let id: Int
+    public let imageUrl: String?
+    public let location: String?
+    public let title: String?
+    public let created: String
+    public let price: String?
+    
+    public init(id: Int, imageUrl: String?, location: String?, title: String?, created: String, price: String?) {
```

Comment:
What's the `id`? Are we sure this will always be unique, even if the same favorite item happen to be added twice in the shelf?
Just making sure since the app front page will crash if not all items are unique 😅  so possibly we should use UUID for uniqueness as well.